### PR TITLE
Fixed discovery tests to use new infra with external libvirt

### PR DESCRIPTION
**Pull request by sghai**
First PR SatelliteQE/automation-tools/pull/271 needs to be merged before this. I'll test this and paste the test results..

### CODE REVIEW

**Comment by sghai** [comment]
```
[sghai@dhcp -robottelo]$ nosetests  -c robottelo.properties  -m test_host_discovery /home/sghai/QE/robottelo/tests/foreman/ui/test_discoveredhosts.py 
.
----------------------------------------------------------------------
Ran 1 test in 217.560s

OK
[sghai@dhcp -robottelo]$ nosetests  -c robottelo.properties  -m test_delete_discovered_host_1 /home/sghai/QE/robottelo/tests/foreman/ui/test_discoveredhosts.py 
.
----------------------------------------------------------------------
Ran 1 test in 249.003s

OK
```

File: tests/foreman/ui/test_discoveredhosts.py [hunk 1]
```diff
@@ -97,7 +110,7 @@ def test_host_discovery(self):
         @Assert: Host should be successfully discovered
 
         """
-        mac = gen_mac(multicast=True, locally=True)
+        mac = gen_mac(multicast=False, locally=True)
```
Comment (kbidarkar) [inline tests/foreman/ui/test_discoveredhosts.py]: Why is this changed to False, just curious ?

File: tests/foreman/ui/test_discoveredhosts.py [hunk 2]
```diff
@@ -14,18 +14,18 @@ class Discovery(UITestCase):
...
-        sleep(30)
+        ssh.command(
+            u'virt-install --hvm --network=bridge:{0}, --mac={1} '
+            '--pxe --name {2} --ram=1024 --vcpus=1 --os-type=linux '
+            '--os-variant=rhel7 --disk path={3},size=10 --noautoconsole'
+            .format(settings.vlan_networking.bridge, mac, self.name,
+                    self.image_path),
+            hostname=self.libvirt_host
+        )
+        sleep(70)
```
Comment (kbidarkar) [inline tests/foreman/ui/test_discoveredhosts.py]: I believe at this stage there is no way for us to know the ip of the "to be discovered host" and hence we can't check whether the "to be discovered host" is up and hence the `sleep` here ? Can you confirm the same please ?

File: tests/foreman/ui/test_discoveredhosts.py (hunk 1 above)
Comment (sghai) [inline tests/foreman/ui/test_discoveredhosts.py]: Sure.. actually we are running tests on external libvirt and there behaviour got changed.. Tests were failing due to below error:

```
robottelo.ssh: DEBUG: <<< stderr
ERROR    XML error: expected unicast mac address, found multicast '03:18:29:2b:ec:e0'
```
So I set the multicast to `False`

File: tests/foreman/ui/test_discoveredhosts.py (hunk 2 above)
Comment (sghai) [inline tests/foreman/ui/test_discoveredhosts.py]: Yeah.. here we are discovering a unknown host and we don't know the IP in advance. So I put a sleep and we increased the timeout from `30 to `70` as we are using another server(external libvirt). Earlier we were using same server via nested virt.

I'm open for any suggestion to get rid of this sleep. thanks for review comments.

**Comment by sthirugn** [comment]
ACK

**Comment by kbidarkar** [comment]
ACK
